Donation-platform/src/main/java/com/google/model/Charity.java

```diff
@@ -86,4 +91,22 @@ public Double getTrendingScoreCharity() {
   public void setTrendingScoreCharity(Double trendingScore) {
     this.trendingScore = trendingScore;
   }
+
```

Comment:
Can we change getCategories() to return a collection of Tag objects? I need the objects themselves for personalization, and I don't think the Keys are useful to Aneri or I (please correct me if I'm wrong on that).

Reply:
resolved. Created function called getTagObjectsbyIds.